[Vaultwarden] does not start without persistence enabled

# Details

**Helm chart name and version:**

vaultwarden 3.1.3

**What steps did you take and what happened:**

helm install vaultwarden k8s-at-home/vaultwarden

**Relevant Helm values:**

persistence.config.enabled: false

**What did you expect to happen:**

I expect that vaultwarden starts with empty dir 'config' created inside the container.

**Anything else you would like to add:**

If I enable persistence, I expect my volumes not to be deleted  if I uninstall the helm chart.

**Logs**

```txt
[2021-11-01 11:13:28.573][vaultwarden][ERROR] Data folder 'config' doesn't exist.
[2021-11-01 11:13:28.573][vaultwarden][ERROR] Verify that your data volume is mounted at the correct location.
```

--- a/charts/stable/vaultwarden/Chart.yaml
+++ b/charts/stable/vaultwarden/Chart.yaml
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.22.2
 description: Vaultwarden is a Bitwarden compatable server in Rust
 name: vaultwarden
-version: 3.2.0
+version: 3.2.1
 kubeVersion: ">=1.16.0-0"
 keywords:
 - Vaultwarden

--- a/charts/stable/vaultwarden/README.md
+++ b/charts/stable/vaultwarden/README.md
@@ -1,6 +1,6 @@
 # vaultwarden
 
-![Version: 3.2.0](https://img.shields.io/badge/Version-3.2.0-informational?style=flat-square) ![AppVersion: 1.22.2](https://img.shields.io/badge/AppVersion-1.22.2-informational?style=flat-square)
+![Version: 3.2.1](https://img.shields.io/badge/Version-3.2.1-informational?style=flat-square) ![AppVersion: 1.22.2](https://img.shields.io/badge/AppVersion-1.22.2-informational?style=flat-square)
 
 Vaultwarden is a Bitwarden compatable server in Rust
 
@@ -69,7 +69,15 @@ helm install vaultwarden k8s-at-home/vaultwarden -f values.yaml
 
 ## Custom configuration
 
-N/A
+The Vaultwarden chart requires the `/config` folder to exist. In order to provide this, some type of storage needs to be implemented.
+For testing purposes, the following config snippet will work:
+
+````yaml
+persistence:
+  config:
+    enabled: true
+    type: emptyDir
+````
 
 ## Values
 
@@ -94,6 +102,16 @@ N/A
 All notable changes to this application Helm chart will be documented in this file but does not include changes from our common library. To read those click [here](https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common#changelog).
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+### [3.2.1]
+
+#### Added
+
+- Mentioned that some kind of persistence is required on `/config` mountpoint.
+
+#### Fixed
+
+- Updated ct-values to properly create emptyDir
 
 ### [3.1.3]
 
@@ -124,6 +142,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Initial version, Succeding Bitwarden_RS.
 
+[3.2.1]: #321
+[3.1.3]: #313
 [3.0.0]: #300
 [2.0.0]: #200
 [1.0.0]: #100

--- a/charts/stable/vaultwarden/README_CHANGELOG.md.gotmpl
+++ b/charts/stable/vaultwarden/README_CHANGELOG.md.gotmpl
@@ -9,6 +9,16 @@ All notable changes to this application Helm chart will be documented in this fi
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [3.2.1]
+
+#### Added
+
+- Mentioned that some kind of persistence is required on `/config` mountpoint.
+
+#### Fixed
+
+- Updated ct-values to properly create emptyDir
+
 ### [3.1.3]
 
 #### Changed
@@ -38,6 +48,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Initial version, Succeding Bitwarden_RS.
 
+[3.1.3]: #313
 [3.0.0]: #300
 [2.0.0]: #200
 [1.0.0]: #100

--- a/charts/stable/vaultwarden/README_CONFIG.md.gotmpl
+++ b/charts/stable/vaultwarden/README_CONFIG.md.gotmpl
@@ -5,5 +5,13 @@
 {{- define "custom.custom.configuration" -}}
 {{ template "custom.custom.configuration.header" . }}
 
-N/A
+The Vaultwarden chart requires the `/config` folder to exist. In order to provide this, some type of storage needs to be implemented.
+For testing purposes, the following config snippet will work:
+
+````yaml
+persistence:
+  config:
+    enabled: true
+    type: emptyDir
+````
 {{- end -}}

--- a/charts/stable/vaultwarden/ci/ct-values.yaml
+++ b/charts/stable/vaultwarden/ci/ct-values.yaml
@@ -1,5 +1,4 @@
 persistence:
   config:
     enabled: true
-    emptyDir:
-      enabled: true
+    type: emptyDir

--- a/charts/stable/vaultwarden/values.yaml
+++ b/charts/stable/vaultwarden/values.yaml
@@ -59,9 +59,6 @@ ingress:
 persistence:
   config:
     enabled: false
-    emptydir:
-      enabled: false
-
 
 # https://github.com/bitnami/charts/tree/master/bitnami/mariadb/#installing-the-chart
 mariadb:

